charmcraft/commands/build.py

```diff
@@ -416,14 +416,8 @@ def pack_charm_in_instance(
             instance_output_dir = env.get_managed_environment_home_path()
             pull_charm = True
 
-        cmd = ["charmcraft", "pack", "--bases-index", str(bases_index)]
-
-        if emit.get_mode() == EmitterMode.VERBOSE:
-            cmd.append("--verbose")
-        elif emit.get_mode() == EmitterMode.QUIET:
-            cmd.append("--quiet")
-        elif emit.get_mode() == EmitterMode.TRACE:
-            cmd.append("--trace")
+        mode = emit.get_mode().name.lower()
+        cmd = ["charmcraft", "pack", "--bases-index", str(bases_index), f"--verbosity={mode}"]
```

Comment:
This is just a personal opinion, but personally I prefer the explicit way.

At least I would add a comment explaining what is being converted to what...

Reply:
Hello! 

Sorry, not following your reasoning. New code is not implicit, but uses the new `--verbosity` option which has the benefit of setting the mode directly without needing to know about or pass different options.

Also, FTR, `--trace` does not exist anymore.